tests/drivers/charger/sbs_charger/testcase.yaml

```diff
@@ -0,0 +1,31 @@
+tests:
+  # section.subsection
+  drivers.sbs_charger.emulated:
+    tags: drivers charger
+    filter: >
+      dt_compat_enabled("sbs,sbs-charger") and
+      (CONFIG_QEMU_TARGET or CONFIG_BOARD_NATIVE_POSIX)
+    extra_args:
+      CONF_FILE="prj.conf;boards/emulated_board.conf"
+      DTC_OVERLAY_FILE="boards/emulated_board.overlay"
+    platform_exclude:
+      qemu_cortex_a53
```

Comment:
We really need to sort this out, can either you or @aaronemassey file an issue so that at least its a tracked thing? This need to split and exclude/allow particular test targets individually based on dts address sizing is pretty unpleasant

Reply:
Ahh I meant to make an issue earlier and let it slip. Made https://github.com/zephyrproject-rtos/zephyr/issues/57356

Thanks, @teburd .